packages/server/src/api/routes/tests/datasource.spec.ts

```diff
@@ -647,6 +647,13 @@ if (datasources.length) {
             // can load it. We're using postgres 16 in tests at the time of writing.
             schema = schema.replace("SET transaction_timeout = 0;", "")
           }
+          if (isPostgres && isLegacy) {
```

Comment:
Should this not be `!isLegacy`, if it's not supported in older versions?

Reply:
In this case its removing removing something from the export - rather than adding something.